src/main/app/claims/models/directionsQuestionnaire.ts

```diff
@@ -0,0 +1,93 @@
+import { YesNoOption } from 'models/yesNoOption'
+
+import { LocalDate } from 'forms/models/localDate'
+import { ReportRow } from 'directions-questionnaire/forms/models/reportRow'
+
+export class DirectionsQuestionnaire {
+
+  constructor (
+    public yesNoOption?: YesNoOption,
+    public howManyOtherWitness?: number,
+    public hearingLocation?: string,
+    public hearingLocationSlug?: string,
+    public exceptionalCircumstancesReason?: string,
+    public unavailableDates?: LocalDate[],
+    public availableDate?: string,
+    public languageInterpreted?: string,
+    public signLanguageInterpreted?: string,
+    public hearingLoopSelected?: YesNoOption,
+    public disabledAccessSelected?: YesNoOption,
+    public otherSupportRequired?: string,
+    public expertReportsRows?: ReportRow[],
+    public expertEvidenceToExamine?: string,
+    public whyExpertIsNeeded?: string
+  ) {
+  }
+
+  deserialize (input: any): DirectionsQuestionnaire {
+    if (input) {
```

Comment:
You will need to have if statements in here as not all of these values may be present so for example input.otherSupportRequired is an optional field so it may not be populated.